src/gatsby/src/styles/search.scss

```diff
@@ -582,6 +582,12 @@ div[class$="-singleValue"] {
       outline: $colorBlueMed auto 1px;
     }
   }
+  ul {
+    padding-left: 32px !important;
```

Comment:
The style from commons.css on the homepage has this as `2rem` and it might be good to use rem here too, so it will scale with the font size. (If that ever happens)

By the way, when I tested this locally, there were parks with every letter including X/Y/Z - did you manually filter something out to test this?

Reply:
I just added `<NoSearchResults />` outside of the condition and checked how it looked like 😂 Changing park names/losing parks is not common, so this message probably won't be seen on the A-Z page but it's always good to have a fallback!